### PR TITLE
Send Pyticketswitch version in User-Agent

### PR DESCRIPTION
**Pull request by adeadman**
This PR extracts the package version to a single file, which is referenced in every place the version string is used (docs, setup.py and within the client itself).

The client uses the version string when it makes a request, and sets it in the custom user agent. This means that clients using pyticketswitch will be identifiable as `pyticketswitch` and not as `python-requests`.

### CODE REVIEW

File: pyticketswitch/__init__.py [hunk 1]
```diff
@@ -1 +1,9 @@
+import json
+from os.path import dirname
 from pyticketswitch.client import Client  # NOQA
+
+
+with open('{}/pkg_info.json'.format(dirname(__file__))) as fp:
+    __pkg_info__ = json.load(fp)
```
Comment (nicwest) [inline pyticketswitch/__init__.py]: I reeeeeeeeeeeeeeally don't like this. it's a massive side effect

**Comment by adeadman** [comment]
@nicwest pkg_info has been removed

**Comment by nicwest** [comment]
:shipit:

File: pyticketswitch/client.py [hunk 2]
```diff
@@ -128,6 +128,18 @@ def get_auth(self):
         if self.user and self.password:
             return (self.user, self.password)
 
+    @property
```
Comment (nicwest) [inline pyticketswitch/client.py]: does this need to be a property?

Comment (adeadman) [reply pyticketswitch/client.py]: You want me to say "no"

Comment (nicwest) [reply pyticketswitch/client.py]: :shipit:

**Pull request merged**